tests/workflows/test_parse_error_message/workflow.py

```diff
@@ -0,0 +1,26 @@
+from vellum.workflows.nodes.bases import BaseNode
+from vellum.workflows.state.base import BaseState
+from vellum.workflows.workflows.base import BaseInputs, BaseWorkflow
+
+
+class Inputs(BaseInputs):
+    value: str
+
+
+class ErrorNode(BaseNode[BaseState]):
+    """A node that raises a generic exception to test _parse_error_message logic."""
+
+    class Outputs(BaseNode.Outputs):
+        result: str
+
+    def run(self) -> Outputs:
+        raise ValueError("Test error from node")
```

Comment:
Update this node to invoke the `execute_integration_tool` under the hood. Then in the test, we will assume a `ForbiddenError` gets raised by the client